lib/plugins/aws/lib/updateStack.js

```diff
@@ -62,6 +62,11 @@ module.exports = {
       params.Parameters = this.serverless.service.provider.stackParameters;
     }
 
+    if (this.serverless.service.provider.disableRollback) {
```

Comment:
I believe similar set of changes should be applied to `createStack` logic, what do you think?

Reply:
Agreed, I added it